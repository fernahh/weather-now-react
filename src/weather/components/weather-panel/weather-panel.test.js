import React from 'react'
import { shallow } from 'enzyme'
import WeatherPanel from './weather-panel'

describe('<WeatherPanel />', () => {
  it('render correctly', () => {
    const wrapper = shallow(<WeatherPanel />)
    expect(wrapper).toMatchSnapshot()
  })
})