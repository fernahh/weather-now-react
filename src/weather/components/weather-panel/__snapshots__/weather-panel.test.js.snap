// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<WeatherPanel /> render correctly 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <WeatherPanel />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <WeatherCardContainer
          city="Urubici"
          country="br"
          showDetails={true}
/>,
        <WeatherCardContainer
          city="Nuuk"
          country="gl"
/>,
        <WeatherCardContainer
          city="Nairobi"
          country="ke"
/>,
      ],
      "className": "weather-panel",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {
          "city": "Urubici",
          "country": "br",
          "showDetails": true,
        },
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {
          "city": "Nuuk",
          "country": "gl",
        },
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {
          "city": "Nairobi",
          "country": "ke",
        },
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <WeatherCardContainer
            city="Urubici"
            country="br"
            showDetails={true}
/>,
          <WeatherCardContainer
            city="Nuuk"
            country="gl"
/>,
          <WeatherCardContainer
            city="Nairobi"
            country="ke"
/>,
        ],
        "className": "weather-panel",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "city": "Urubici",
            "country": "br",
            "showDetails": true,
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "city": "Nuuk",
            "country": "gl",
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "city": "Nairobi",
            "country": "ke",
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getSnapshotBeforeUpdate": true,
        },
      },
    },
    "attachTo": undefined,
    "hydrateIn": undefined,
  },
}
`;
